migrate.py

```diff
@@ -5,6 +5,7 @@
 from pathlib import Path
 from sqlite3 import connect
 from shutil import move as fsmove
+import sqlite3
```

Comment:
With dropping `cursor` (and maybe execute_sql entirely), can remove this import. Also, you could've added `Cursor` as an import from sqlite3.